packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -296,6 +300,16 @@ class InteractiveViewer extends StatefulWidget {
   /// than maxScale.
   final double minScale;
 
+  /// Changes the scrolling behavior after end of a gesture.
+  ///
+  /// Used as the coefficient of friction in the inertial translation animation.
+  /// This value was eyeballed to give a feel similar to Google Photos.
+  ///
+  /// Defaults to 0.0000135.
+  ///
+  /// Cannot be null, and must be a finite number greater than zero
```

Comment:
Can we add documentation on how the scroll changes as the value increases or decreases?